data/tr1/ship/cfg/TR1X_strings.json5

```diff
@@ -0,0 +1,521 @@
+                "key_4": {"name": "Thor Key"},
+            },
+        },
+
+        // Level 6: Colosseum
+        {
+            "title": "Colosseum",
+            "objects": {
+                "key_1": {"name": "Rusty Key"},
+            },
+        },
+
+        // Level 7: Palace Midas
+        {
+            "title": "Palace Midas",
+            "objects": {
+                "puzzle_1": {"name": "Gold Bar"},
+            },
+        },
+
+        // Level 8: The Cistern
+        {
+            "title": "The Cistern",
+            "objects": {
+                "key_1": {"name": "Gold Key"},
+                "key_2": {"name": "Silver Key"},
+                "key_3": {"name": "Rusty Key"},
+            },
+        },
+
+        // Level 9: Tomb of Tihocan
+        {
+            "title": "Tomb of Tihocan",
+            "objects": {
+                "key_1": {"name": "Gold Key"},
+                "key_2": {"name": "Rusty Key"},
+                "key_3": {"name": "Rusty Key"},
+            },
+        },
+
+        // Level 10: City of Khamoon
+        {
+            "title": "City of Khamoon",
+            "objects": {
+                "key_1": {"name": "Sapphire Key"},
+            },
+        },
+
+        // Level 11: Obelisk of Khamoon
+        {
+            "title": "Obelisk of Khamoon",
+            "objects": {
+                "key_1": {"name": "Sapphire Key"},
+                "puzzle_1": {"name": "Eye of Horus"},
+                "puzzle_2": {"name": "Scarab"},
+                "puzzle_3": {"name": "Seal of Anubis"},
+                "puzzle_4": {"name": "Ankh"},
+            },
+        },
+
+        // Level 12: Sanctuary of the Scion
+        {
+            "title": "Sanctuary of the Scion",
+            "objects": {
+                "key_1": {"name": "Gold Key"},
+                "puzzle_1": {"name": "Ankh"},
+                "puzzle_2": {"name": "Scarab"},
+            },
+        },
+
+        // Level 13: Natla's Mines
+        {
+            "title": "Natla's Mines",
+            "objects": {
+                "puzzle_1": {"name": "Fuse"},
+                "puzzle_2": {"name": "Pyramid Key"},
+            },
+        },
+
+        // Level 14: Atlantis
+        {
+            "title": "Atlantis",
+        },
+
+        // Level 15: The Great Pyramid
+        {
+            "title": "The Great Pyramid",
+        },
+
+        // Level 16: Cut Scene 1
+        {
+            "title": "Cut Scene 1",
+        },
+
+        // Level 17: Cut Scene 2
+        {
+            "title": "Cut Scene 2",
+        },
+
+        // Level 18: Cut Scene 3
+        {
+            "title": "Cut Scene 3",
+        },
+
+        // Level 19: Cut Scene 4
+        {
+            "title": "Cut Scene 4",
+        },
+
+        // Level 20: Title
+        {
+            "title": "Title",
+        },
+
+        // Level 21: Current Position
+        {
+            "title": "Current Position",
+        },
+    ],
```

Comment:
Looks good overall. Only obstacle I foresee for builders is having to basically do some duplication between the gameflow and strings file. Both files will need to be updated and make sure the number of levels match. But overall, I think this is a big improvement for the future and will make it easier to update.